conf/script/build-cortx-ha

```diff
@@ -20,7 +20,17 @@ set -eu -o pipefail
 
 HA_PATH="<HA_PATH>"
 
+log_file=${BUILD_CORTX_HA_LOG_FILE:-/var/log/build-cortx-ha.log}
+exec &>> $log_file
+exec &> >(stdbuf -oL gawk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0 }')
+
+log() {
+    logger --stderr --tag build-cortx-ha "$*"
+}
+
+
 init() {
+    log "${FUNCNAME[0]}: Init cluster"
```

Comment:
How do we control logging level?

Reply:
It is deployment stage. I think logging is not supposed to be controlled somehow during provisioning.

Reply:
It is a good point for low-level HA requirements discussion. Runtime logging shall be controllable for sure. But deployment - i see not use case in that so far.

Reply:
This log  is added for the sake of completeness; so that every stage gets reported in the logs. 
We can remove it, if that is the general consensus

we are currently using the linux logger utility for logging here. 
I have been unable to find any way to configure logging level using the linux logger. 
>>How do we control logging level?
So not sure how this can be achieved even after deployment stage.

Reply:
ok, understood.